tests/functional/behave_features/HC-18_multiple_charts_in_pr.feature

```diff
@@ -0,0 +1,113 @@
+Feature: Multiple charts submission in one PR
+    If partners, redhat and community users try to submit multiple charts in one single PR
+    they will get an appropriate error message
+
+    Scenario Outline: [HC-18-001] A user submits a PR with multiple reports
+        Given the vendor "<vendor>" has a valid identity as "<vendor_type>"
+        And user wants to send two reports as in "<report_path_1>" and "<report_path_2>"
+        When the user sends a pull request with the report
+        Then the pull request is not merged
+        And user gets the "<message>" in the pull request comment
+    
+        @partners @full
```

Comment:
These test should fail fast, would like to see just one test in smoke.

Reply:
Added one test in the smoke now.